Neos.Media/Classes/Domain/Model/ThumbnailGenerator/ImageThumbnailGenerator.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
uhm no we don't, just checked 10 random ones and they all didn't.. maybe you're using some kind of special rules normally?